pyface/i_widget.py

```diff
@@ -35,6 +35,9 @@ class IWidget(Interface):
     #: A tooltip for the widget.
     tooltip = Str()
 
+    #: An optional context menu for the widget.
+    context_menu = Instance("pyface.action.menu_manager.MenuManager")
```

Comment:
What about the `show_context_menu` method that is defined on the `IField` interface? Update : It looks like an undefined/unused part of the `IField` interface which we're choosing not to carry over.

Reply:
Yes, I realized it wasn't hooked up to anything, so no reason to carry it over.